tests/slow/src/test/scala/tests/feature/FoldingCrossLspSuite.scala

```diff
@@ -0,0 +1,43 @@
+package tests.feature
+
+import tests.BaseLspSuite
+import scala.meta.internal.metals.{BuildInfo => V}
+
+class FoldingCrossLspSuite extends BaseLspSuite("foldingRange-cross") {
```

Comment:
Question: Is this test related to mentioned optimisation removal?

Reply:
Yes actually, this issue came up when using Folding on Scala versions different than `2.12.11`